app/src/main/res/values-en/strings.xml

```diff
@@ -998,7 +998,9 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_surface_detailed_answer_impossible_title">Describe surface</string>
     <string name="open_url">Open URL</string>
     <string name="quest_dietType_kosher_name_title">Does %s offer kosher products?</string>
+    <string name="quest_dietType_halal_name_title">Does %s offer halal products?</string>
     <string name="quest_dietType_explanation_kosher">Kosher products must be prepared according to special rules, with kashrut certification agencies marking products in compliance.</string>
+    <string name="quest_dietType_explanation_halal">Halal products must be prepared according to special rules, with Halal certification agencies marking products in compliance.</string>
```

Comment:
So this is really the same as with kashrut? Are there "certification agencies" what mark products in compliance? Or is it enough if there is a label somewhere that something is halal?

Reply:
It looks like it based on https://en.wikipedia.org/wiki/Halal_certification_in_Europe